src/Units/KaijuWarsUnits.java

```diff
@@ -0,0 +1,791 @@
+package Units;
+
+import java.util.ArrayList;
+
+import Engine.GameInstance;
+import Engine.UnitActionFactory;
+import Engine.GameEvents.GameEventQueue;
+import Engine.GameEvents.HealUnitEvent;
+import Engine.GameEvents.ResupplyEvent;
+import Engine.StateTrackers.StateTracker;
+import Engine.StateTrackers.SuicideAttackTracker;
+import Engine.StateTrackers.UnitResurrectionTracker;
+import Engine.StateTrackers.UnitTurnPositionTracker;
+import Engine.UnitActionLifecycles.UnitProduceLifecycle;
+import Terrain.MapMaster;
+import Terrain.TerrainType;
+import Terrain.Environment.Weathers;
+import Units.MoveTypes.Flight;
+import Units.MoveTypes.FootMech;
+import Units.MoveTypes.MoveType;
+import Units.MoveTypes.Tread;
+
+public class KaijuWarsUnits extends UnitModelScheme
+{
+  private static final long serialVersionUID = 1L;
+
+  @Override
+  public String toString()
+  {
+    return super.toString() + "Kaiju Wars";
+  }
+
+  @Override
+  public String getIconicUnitName()
+  {
+    // TODO: Does the infrastructure around this need to change, or should this go away?
+    return "Infantry";
+  }
```

Comment:
Is this TODO still relevant?

Reply:
Yes, but probably out of scope for this PR